corehq/apps/reports/dbaccessors.py

```diff
@@ -1,56 +1,8 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
-import json
-
-from django.conf import settings
-
 from corehq.apps.domain.dbaccessors import get_docs_in_domain_by_class
 
 
-def _get_exports(domain, include_docs=True, reduce=False, **kwargs):
-    from corehq.apps.reports.models import HQExportSchema
-    # add saved exports. because of the way in which the key is stored
-    # (serialized json) this is a little bit hacky, but works.
-    startkey = json.dumps([domain, ""])[:-3]
-    endkey = "%s{" % startkey
-    return HQExportSchema.get_db().view(
```

Comment:
There are no more `SavedExportSchema` documents, meaning that this view will always return nothing.  There's a bunch more to delete, this is just an easy to break out start